DopplerCurrencyJob/Exceptions/DopplerCurrencyJobException.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Doppler.Currency.Job.Exceptions
+{
+    public class DopplerCurrencyJobException : Exception
```

Comment:
Why don't use directly Exception? I think that this class is unnecessary if not has properties declared

Reply:
I understand ..., but it is a way to identify that it is a manually thrown exception. In addition, it leaves open the possibility of personalizing it. But ultimately you are right, how is it done is unnecessary.

Reply:
Entiendo que se decidió que quede la excepción personalizada, cierro la convesación.